src/util-log-redis.c

```diff
@@ -514,7 +515,9 @@ int SCConfLogOpenRedis(ConfNode *redis_node, void *lf_ctx)
         SCLogError(SC_ERR_MEM_ALLOC, "Error allocating redis server string");
         exit(EXIT_FAILURE);
     }
-    log_ctx->redis_setup.port = atoi(redis_port);
+    if (StringParseU16RangeCheck(&log_ctx->redis_setup.port, 10, 0, (const char *)redis_port, 0, 65535) < 0) {
+        FatalError(SC_ERR_INVALID_VALUE, "Invalid value for redis port");
```

Comment:
Suggest including the invalid value in this message `"Invalid redis port value '%s'"`